### PR TITLE
Fix MongoDB index for GetRunnableInstances

### PR DESCRIPTION
**Pull request by glucaci**
**Describe the change**
GetRunnableInstances needs both `NextExecution` and `Status` for the MongoDB query.

**Describe your implementation or design**
Change the current index and rename it to don't conflict on creation. The old one must be deleted manually.

**Breaking change**
No

### CODE REVIEW

**Comment by glucaci** [comment]
PostgresSQL tests seams to have a problem
`Cannot write DateTime with Kind=Local to PostgreSQL type 'timestamp with time zone', only UTC is supported`

**Comment by glucaci** [comment]
I did also some changes regarding the tests because it was not working anymore:
- change container support to `Squadron`
- run tests in parallel to see exactly what is failing
- consolidate dependencies

**Comment by glucaci** [comment]
@danielgerlag do you have any time to review this one?

**Comment by glucaci** [comment]
Just fixed the Postgres tests by enabeling legacy timestamp behaviour **only in tests**
```
AppContext.SetSwitch("Npgsql.EnableLegacyTimestampBehavior", true);
```

It will be nice to put it somewhere in docu if the usres are migrating to the newest postgres and have the `Timestamp` problem to know what to do.

**Review by danielgerlag — APPROVED** [review]

File: src/providers/WorkflowCore.Persistence.MongoDB/Services/MongoPersistenceProvider.cs [hunk 1]
```diff
@@ -93,8 +93,10 @@ static void CreateIndexes(MongoPersistenceProvider instance)
             if (!indexesCreated)
             {
                 instance.WorkflowInstances.Indexes.CreateOne(new CreateIndexModel<WorkflowInstance>(
-                    Builders<WorkflowInstance>.IndexKeys.Ascending(x => x.NextExecution),
-                    new CreateIndexOptions {Background = true, Name = "idx_nextExec"}));
+                    Builders<WorkflowInstance>.IndexKeys
```
Comment (murtoza) [inline src/providers/WorkflowCore.Persistence.MongoDB/Services/MongoPersistenceProvider.cs:96]: In our experiment, we found that the following index works better 

```
 instance.WorkflowInstances.Indexes.CreateOne(new CreateIndexModel<WorkflowInstance>(
                    Builders<WorkflowInstance>.IndexKeys
                        .Ascending(x => x.NextExecution)
                        .Ascending(x => x.Status)
                        .Ascending(x => x.Id),
                    new CreateIndexOptions {Background = true, Name = "idx_nextExec_Status_Id)"}));
```

Comment (glucaci) [reply src/providers/WorkflowCore.Persistence.MongoDB/Services/MongoPersistenceProvider.cs:96]: Nice catch, thanks!
Just did a test and both run on index but I missed the projection.
image

**Comment by glucaci** [comment]
@danielgerlag thanks for approving, I think is ready to be merged.

File: test/WorkflowCore.Tests.MySQL/WorkflowCore.Tests.MySQL.csproj [hunk 2]
```diff
@@ -1,9 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
-    <PackageReference Include="xunit" Version="2.4.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <PackageReference Include="Squadron.MySql" Version="0.17.0" />
```
Comment (danielgerlag) [inline test/WorkflowCore.Tests.MySQL/WorkflowCore.Tests.MySQL.csproj:4]: @glucaci  
All of the MySQL tests are failing for me - `MySqlException - no database selected`

**Pull request merged**